src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -1,13 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../build-common/NHibernate.props" />
   <PropertyGroup>
     <Description>The Unit Tests for NHibernate.</Description>
     <TargetFrameworks>$(NhAppTargetFrameworks)</TargetFrameworks>
     <IsTestProject>true</IsTestProject>
-    <NoWarn>$(NoWarn);3001;3002;3003;3005</NoWarn>
+    <NoWarn>$(NoWarn);3001;3002;3003;3005;SYSLIB0003;SYSLIB0011;SYSLIB0012</NoWarn>
     <ContentSQLiteInteropFiles>true</ContentSQLiteInteropFiles>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
```

Comment:
Why not introduce global `NhNetCore` property and use it for all such places instead to simplify further retargeting?

Reply:
Because this is not so easy... I tried it when targeting both .NET Core 3.1 and .NET 6... The dependencies could/were a little different. For example, `Microsoft.AspNetCore.OData` does not have problems on 3.1 but has problems in 6.0.

Reply:
Ehm...  I want to convert checks like `<PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">` to something like `<PropertyGroup Condition="NhNetCore">`.  Looks easy enough for me.. Not sure I understand your problem - are you talking about targeting both 3.1 and 6.0? IMO that's a bit different issue and should be handled separately

Reply:
> Looks easy enough

Yes, it is easy, but I doubt it will solve any real issue.

> are you talking about targeting both 3.1 and 6.0?

I was going to make it target .NET 3.1 and .NET 6 rc initially. I started working on the switch around last betas and before .NET 6 rc.1 came out. But I did not have enough time and motivation to finish that at that time. So because of differences in the supported features (like SYSLIB0011 - BinaryFormatter depreciation which is quite aggressive) and different dependencies between .NET 3.1 and .NET 6.0 it required having 2 different monikers like NhNetCore31 and NhNet60. Which defied the purpose of the simplification. 

Also, the same could happen when we transition from .NET 6 to next version. Depends on the path we're willing to take.  

So, frankly, I do not believe that a simple bump of a version in NhNetCore would be enough to migrate to the later versions. Seems like we're trying to solve the problem which we might not have. 

For NhNet it does have sense to use it because Microsoft has promised that 4.x API would stay backwards compatible.